config/build/rollup.config.js

```diff
@@ -225,10 +162,7 @@ const createConfig = (
                 },
             }),
             babel({
-                babelHelpers:
-                    platform === "browser" && format === "esm"
-                        ? "runtime"
-                        : "bundled",
+                babelHelpers: "runtime",
```

Comment:
WonderBlocks uses [`runtime`](https://github.com/Khan/wonder-blocks/blob/9bbbb01d19e7da25ad7eadc30883e73b2f1473f9/build-settings/rollup.config.mjs#L44) for both ES and CJS builds, so we'll do the same!

https://www.npmjs.com/package/@rollup/plugin-babel#babelhelpers